tests/integration/cgroups.bats

```diff
@@ -176,3 +178,37 @@ function setup() {
     [ "$status" -eq 0 ]
 #
 }
+
+@test "runc run (cgroup v1 + unified resources should fail)" {
+    requires root cgroups_v1
+
+    set_cgroups_path "$BUSYBOX_BUNDLE"
+    set_resources_limit "$BUSYBOX_BUNDLE"
+    update_config '.linux.resources.unified |= {"memory.min": "131072"}' "$BUSYBOX_BUNDLE"
+
+    runc run -d --console-socket "$CONSOLE_SOCKET" test_cgroups_unified
+    [ "$status" -ne 0 ]
+    [[ "$output" == *'invalid configuration'* ]]
+}
+
+@test "runc run (cgroup v2 + unified resources)" {
```

Comment:
Can we also have test for pure-unified config

Reply:
will do

Reply:
added; PTAL